servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/ErrorContext.java

```diff
@@ -3,25 +3,24 @@
 
 package com.microsoft.azure.servicebus.primitives;
 
+import java.io.Serializable;
 import java.util.Locale;
 
-abstract class ErrorContext
-{
+abstract class ErrorContext implements Serializable {
+
+    private static final long serialVersionUID = -6342329018037308640L;
```

Comment:
[SpotBugs]
Since ServiceBusException has serialVersionUID, it means it needs to implement Serializable.  Change the ErrorContext to implements Serializable cause all its sub-extended class to have the serialVersionUID